### PR TITLE
Adding remaining functions, writing cucumber steps.

### PR DESCRIPTION
**Pull request by open-schnick**

### CODE REVIEW

File: src/test/java/de/filefighter/rest/cucumber/CommonCucumberSteps.java [hunk 1]
```diff
@@ -1,34 +1,98 @@
...
 
     // file / folder
     @Given("{string} exists with id {long} and path {string}")
-    public void existsWithIdAndPath(String fileOrFolder, long fsItemId, String arg2) {
+    public void existsWithIdAndPath(String fileOrFolder, long fsItemId) {
+        if(fileOrFolder.equals("file")){
+            fileSystemRepository.save(FileSystemEntity
+                    .builder()
+                    .isFile(true)
+                    .id(fsItemId)
```
Comment (qvalentin) [inline src/test/java/de/filefighter/rest/cucumber/CommonCucumberSteps.java:74]: why is path ignored?

**Review by qvalentin — COMMENTED** [review]
writing stepfunctions seem quite easy actually with the whole springboot setup

File: src/test/java/de/filefighter/rest/cucumber/CommonCucumberSteps.java (hunk 1 above)
Comment (open-schnick) [reply src/test/java/de/filefighter/rest/cucumber/CommonCucumberSteps.java:80]: this is wip.

Comment (open-schnick) [reply src/test/java/de/filefighter/rest/cucumber/CommonCucumberSteps.java:80]: nvm @qvalentin this is fixed one commit later :D

**Comment by open-schnick** [comment]
@qvalentin yes it should be really easy. I implemented in the last commit a possibility to consume the endpoints, and share the result between step files.

**Review by qvalentin — APPROVED** [review]

**Pull request merged**